tests/services/test_feed_manager.py

```
@@ -0,0 +1,84 @@
+import pytest
+from unittest.mock import patch, mock_open
+from pathlib import Path
+from src.services.feed_manager import FeedManager
+from src.models.feed import Feed
+from datetime import datetime
```

Ruff (0.8.0)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)